src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonIListConverter.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    /// Converter for <cref>System.Collections.IList</cref>.
+    internal sealed class JsonIListConverter<TCollection> : JsonIEnumerableDefaultConverter<TCollection, object>
+        where TCollection : IList
+    {
+        protected override void Add(object value, ref ReadStack state)
```

Comment:
Make similar changes throughout.

```suggestion
        protected override void Add(object? value, ref ReadStack state)
```